docker-compose.yaml

```diff
@@ -23,7 +23,7 @@ services:
         depends_on:
           - db
         environment:
-          ROCKET_DATABASE_URL: mysql://test:test@db:3306/pushbox
+          ROCKET_DATABASE_URL: mysql://test:test@db:4306/pushbox
```

Comment:
Is this some change in default behaviour?

Reply:
No, it actually fixes the docker-compose to use the specified database. On line 9 above, we export `4306:3306`
https://github.com/mozilla-services/pushbox/blob/b93d9183cccac835912fbc0cb19cbf6cfe23ff35/docker-compose.yaml#L9
This just makes sure that the database URL uses that port.